lib/shoulda/matchers/active_model/qualifiers/ignoring_interference_by_writer.rb

```diff
@@ -6,7 +6,7 @@ module Qualifiers
         module IgnoringInterferenceByWriter
           attr_reader :ignore_interference_by_writer
 
-          def initialize(*args)
+          def initialize(*_args)
```

Comment:
Since the name isn't important here, if there is a way to enforce this in Rubocop, can we just spell this is as `*`? If not, then no big deal — I don't want you to have to change all of these. Just wondering.

Reply:
I will change this.